app.py

```diff
@@ -27,7 +27,7 @@
 from thoth.common import init_logging
 
 __title__ = "thoth.mi-scheduler"
-__version__ = "1.0.3"
+__version__ = "1.0.2"
```

Comment:
this should be changed automatically by bot, we don't change it manually, maybe rebase?

Reply:
hmm, weird - I've got the 1.0.3 commit on my master and also in this
turns out this was probably an issue with vscode - it had somehow overwritten the newest version with an older version of file
i guess